content/user/payment/square.md

```diff
@@ -7,16 +7,44 @@ weight: 10
 
 There are two Square modules for Zen Cart: 
 - A newer one, based on the Square Web Payments API 
-- An older one, based on SqPaymentForm, which will be deprecated in 2022
+- An older one, based on SqPaymentForm, which will be deprecated in July 2022.
 
 Square users will need to move to the new [Square Web Payments API plugin](https://www.zen-cart.com/downloads.php?do=file&id=2345). 
 
 ## Newer Integration - Square API 
 The newer Square integration is available in the Plugins library as [Square Web Payments API plugin](https://www.zen-cart.com/downloads.php?do=file&id=2345). 
 
-Please note that the newer integration requires PHP 7.4, so you will need
-to be running Zen Cart 1.5.7 or higher. If you need to upgrade, please 
-review the [upgrading instructions](/user/upgrading/). 
+Please note that, while Square suggests PHP 7.4 or greater, we have found this mod will work with PHP 7.3 and Zen Cart version 1.5.6c. That said, upgrading to the current version of Zen Cart and at least PHP 7.4 is recommended for security. If you need to upgrade, please review the [upgrading instructions](/user/upgrading/). 
+
+### Setup Instructions: 
+
+a) On Your Site:
+
+- Go to your admin >> Modules >> Payment and click on the circled-I next to Square.
+Using something like Notepad, copy the Application ID and Application Secret (OAuth) from the existing Square Module. You will need them for the WebPay as well.
+
+
+b) On the Square Website:
+
+- Log in to Square, go to App Integrations, and select manage for your Application
+- Just above Credentials, Click on Production to get to where you need to work
+- Scroll Down to Production API Version and make sure it is the latest
```

Comment:
i am using the using API version: 17.2.0.20220216, which is listed on the square website as:

 2022-02-16 (1 version behind)